openfisca_uk/variables/finance/benefit/family/pension_credit.py

```diff
@@ -137,6 +136,7 @@ def formula(benunit, period, parameters):
         exempted_personal_benefits = aggr(
             benunit, period, EXEMPTED_PERSONAL_BENEFITS
         )
+        # This makes me think we need to push the minimum benefit back to CTC+WTC
```

Comment:
Reverted to draft because I think this is more complicated. @nikhilwoodruff do you agree that provisions like these in other formulas indicates we need an intermediate pre-minimum variable?

Reply:
Oh good catch. Yes, I think that's necessary for it now

Reply:
Done